docs/installation/upgrade/include/2.4-removed-other-features.inc

```diff
@@ -0,0 +1,28 @@
+| Dovecot director role | Replaced with the [Dovecot Pro Palomar architecture](https://doc.dovecotpro.com/). |
+| Global ACL directory | Use [[link,acl_global_file]] instead. See [below](#use-global-acl-files-instead-of-global-acl-directories) for details on migration. |
+| IMAP SETQUOTA command | Quota limits can no longer be modified using the IMAP SETQUOTA command. The `set_quota` setting has been removed. |
+| IPC process | Has been merged to anvil. |
+| OpenSSL support for older than 1.0.2 | Older versions are not supported anymore. |
+| Sieve extensions: `notify`, `imapflags`, `vnd.dovecot.duplicate` | These deprecated Sieve extensions have been removed. |
+| "size.virtual" index record | size.virtual field is no longer written to dovecot.index.cache file as it is duplicating vsize record in dovecot.index file. Reading of the field from old files is supported. |
+| `ssl-parameters.dat` | This file is no longer converted automatically by config process, you need to set [[setting,ssl_dh_file]] setting if you need non-ECC Diffie-Hellman. |
+| TCP wrapper support | Use [[link,auth_lua]] instead. |
+| Weak password schemes | Weak password schemes are disabled by default; you need to use [[setting,auth_allow_weak_schemes]] to enable them. |
+| Global ACL directory | See [ACLs](#acls) below. |
+
+#### dict Proxy Parameters
+
+See [[link,dict_proxy]].
+
+| Parameter | Notes |
+| --------- | ----- |
+| `idle_msecs` | Deprecated dict proxy parameter. Use [[link,dict_idle_timeout]] setting instead. |
+| `warn_slow_msecs` | Deprecated dict proxy parameter. Use [[link,dict_slow_warn]] setting instead. |
```

Comment:
Seems a weird way to say these. The parameters were converted to individual settings, not deprecated. Not really sure how we're going to do these in general, are we going to list all the converted settings.. Also seems wrong to have these under "removed" section, since they weren't really removed.